week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,71 @@
-function setAlarm() {}
+function setAlarm() {
+  //get the value of the input field
+  let inputField = document.querySelector("#alarmSet").value;
+
```

Comment:
You need to do some input validation here. What happens if the input field doesn't have a value or it's unsuitable - e.g. it's a negative number or has letters or spaces?

Reply:
Thanks for your feedback. I have added function for input validation. It informs that user inputted wrong value and reload web page.  Thanks for your assistance.